File: Java/프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        
        for(int i=0; i<rsp.length(); i++) {
            char g = rsp.charAt(i);
            
            if(g == '0') {
                answer += 5;
            }
            else if(g == '2') {
                answer += 0;
            }
            else
                answer += 2;
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = '';
    let arr = [];
    for(let i=0; i<rsp.length; i++) {
        if(rsp.charAt(i) == 2)
            arr.push(0);
        else if (rsp.charAt(i) == 0)
            arr.push(5);
        else
            arr.push(2);
        answer = arr.join('');
    }
    return answer;
}